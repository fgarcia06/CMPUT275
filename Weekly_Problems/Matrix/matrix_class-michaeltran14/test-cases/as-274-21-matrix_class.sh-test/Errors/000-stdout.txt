/home/michaeltran14/CMPUT_275/C++/Weekly_Problems/Matrix/matrix_class-michaeltran14/soln/.build/build.sh: line 9:  3577 Segmentation fault      ./$NAME

Expected file "test-cases/as-274-21-matrix_class.sh-test/Expected/000-stdout.txt" has no output file match
