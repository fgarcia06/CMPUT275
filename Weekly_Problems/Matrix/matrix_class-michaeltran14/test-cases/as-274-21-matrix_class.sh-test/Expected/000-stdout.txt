1 1 1
1 1 1
2 2 2
2 2 2
1 2
3 4
5 6
3 3 3
3 3 3
-1 -1 -1
-1 -1 -1
18 18
18 18
-2 -2 -2
-2 -2 -2
0 1 1
1 1 1
3
1 2 2
2 2 2
0 1
1 1
1 1
0 0
0 0
0 0
3 4
5 6
7 8
